indexer/Cargo.toml

```diff
@@ -35,3 +37,4 @@ near-jsonrpc-client = "0.4.0-beta.0"
 near-lake-framework = "0.5.2"
 
 explorer-database = { path = "../database" }
+http = "0.2.8"
```

Comment:
Would you mind moving it before the `itertools` declarations, please? This is the place for path-dependencies